### DISCUSSION TITLE
Can minio offer resumable upload file?

### DISCUSSION THREAD

**Discussion by beckhamaaa** [minio/minio#12394] [Q&A] [answered]
Hi, i want to know that the minio offer the resumable upload file, it is based the amazon s3 resumable upload protocol, thanks a lot! i use s3cmd as a client.

**Comment by klauspost** [comment, marked as answer]
@beckhamaaa Yes, uploads can be resumed when doing multipart uploads.

Implementing this on the client (s3cmd here) is quite tricky, however. So my guess is that most clients do not use this feature between invocations. Ask s3cmd authors or consult the documentation.

**Comment by aisensiy** [comment]
Is the minio client support this feature? I see there is a `--continue` parameter in `mc cp` but I do not know how to use it and there is no much detail about this parameter.

**Reply by klauspost** [reply]
As answered elsewhere, no. `--continue` is for sessions of multiple files, not individual files.

**Reply by aisensiy** [reply]
Got it. Thanks.
